*** config.plist	Thu Nov 26 15:13:07 2020
--- fallback.plist	Thu Nov 26 15:16:42 2020
***************
*** 303,309 ****
  			<key>ThirdPartyDrives</key>
  			<false/>
  			<key>XhciPortLimit</key>
! 			<false/>
  		</dict>
  		<key>Scheme</key>
  		<dict>
--- 303,309 ----
  			<key>ThirdPartyDrives</key>
  			<false/>
  			<key>XhciPortLimit</key>
! 			<true/>
  		</dict>
  		<key>Scheme</key>
  		<dict>
***************
*** 326,338 ****
  			<key>HibernateMode</key>
  			<string>None</string>
  			<key>HideAuxiliary</key>
! 			<true/>
  			<key>PickerAttributes</key>
  			<integer>1</integer>
  			<key>PickerAudioAssist</key>
  			<false/>
  			<key>PickerMode</key>
! 			<string>External</string>
  			<key>PollAppleHotKeys</key>
  			<false/>
  			<key>ShowPicker</key>
--- 326,338 ----
  			<key>HibernateMode</key>
  			<string>None</string>
  			<key>HideAuxiliary</key>
! 			<false/>
  			<key>PickerAttributes</key>
  			<integer>1</integer>
  			<key>PickerAudioAssist</key>
  			<false/>
  			<key>PickerMode</key>
! 			<string>Builtin</string>
  			<key>PollAppleHotKeys</key>
  			<false/>
  			<key>ShowPicker</key>
***************
*** 340,355 ****
  			<key>TakeoffDelay</key>
  			<integer>0</integer>
  			<key>Timeout</key>
! 			<integer>3</integer>
  		</dict>
  		<key>Debug</key>
  		<dict>
  			<key>AppleDebug</key>
! 			<false/>
  			<key>ApplePanic</key>
! 			<false/>
  			<key>DisableWatchDog</key>
! 			<false/>
  			<key>DisplayDelay</key>
  			<integer>0</integer>
  			<key>DisplayLevel</key>
--- 340,355 ----
  			<key>TakeoffDelay</key>
  			<integer>0</integer>
  			<key>Timeout</key>
! 			<integer>5</integer>
  		</dict>
  		<key>Debug</key>
  		<dict>
  			<key>AppleDebug</key>
! 			<true/>
  			<key>ApplePanic</key>
! 			<true/>
  			<key>DisableWatchDog</key>
! 			<true/>
  			<key>DisplayDelay</key>
  			<integer>0</integer>
  			<key>DisplayLevel</key>
***************
*** 359,365 ****
  			<key>SysReport</key>
  			<false/>
  			<key>Target</key>
! 			<integer>0</integer>
  		</dict>
  		<key>Entries</key>
  		<array/>
--- 359,365 ----
  			<key>SysReport</key>
  			<false/>
  			<key>Target</key>
! 			<integer>67</integer>
  		</dict>
  		<key>Entries</key>
  		<array/>
***************
*** 437,443 ****
  				<key>prev-lang:kbd</key>
  				<data></data>
  				<key>boot-args</key>
! 				<string>keepsyms=1</string>
  				<key>csr-active-config</key>
  				<data>fwAAAA==</data>
  			</dict>
--- 437,443 ----
  				<key>prev-lang:kbd</key>
  				<data></data>
  				<key>boot-args</key>
! 				<string>-v keepsyms=1 debug=0x12a acpi_layer=0x08 acpi_level=0x02</string>
  				<key>csr-active-config</key>
  				<data>fwAAAA==</data>
  			</dict>
***************
*** 583,595 ****
  			<key>ReconnectOnResChange</key>
  			<false/>
  			<key>ReplaceTabWithSpace</key>
! 			<true/>
  			<key>Resolution</key>
  			<string>Max</string>
  			<key>SanitiseClearScreen</key>
  			<false/>
  			<key>TextRenderer</key>
! 			<string>SystemGraphics</string>
  			<key>UgaPassThrough</key>
  			<false/>
  		</dict>
--- 583,595 ----
  			<key>ReconnectOnResChange</key>
  			<false/>
  			<key>ReplaceTabWithSpace</key>
! 			<false/>
  			<key>Resolution</key>
  			<string>Max</string>
  			<key>SanitiseClearScreen</key>
  			<false/>
  			<key>TextRenderer</key>
! 			<string>BuiltinGraphics</string>
  			<key>UgaPassThrough</key>
  			<false/>
  		</dict>
